src/platform/plugins/shared/data_views/common/data_views/abstract_data_views.ts

```diff
@@ -388,6 +393,7 @@ export abstract class AbstractDataView {
       runtimeFieldMap: stringifyOrUndefined(this.runtimeFieldMap),
       name: this.name,
       allowHidden: this.allowHidden,
+      // managed: this.managed,
```

Comment:
Do we want to pass the managed property here?

Reply:
removed!